processing/src/test/java/org/apache/druid/query/operator/window/RowsAndColumnsHelper.java

```diff
@@ -280,6 +281,17 @@ public void validate(String msgBase, Column col)
           } else {
             Assert.assertEquals(msg, ((Long) expectedVal).longValue(), accessor.getLong(i));
           }
+        } else if (expectedVal instanceof Object[]) {
+          Object actualVal = accessor.getObject(i);
+          if (expectedNulls[i]) {
+            Assert.assertEquals(msg, 0, actualVal);
```

Comment:
Question: Does this check mean that if the row value is null (in expected), then actual val should be 0?

Reply:
Thanks for pointing out. it was a mistake. Should have been `Assert.assertNull(msg, accessor.getObject(i))`
will update it.